src/commands/dev/dev.ts

```diff
@@ -1,6 +1,5 @@
 import process from 'process'
 
-// @ts-expect-error TS(7016) FIXME: Could not find a declaration file for module '@net... Remove this comment to see the full error message
 import { applyMutations } from '@netlify/config'
```

Comment:
This is fixed by this:
https://github.com/netlify/build/pull/5692/commits/cf459316c28d49444330593fcc393c0febc80cb0